test/model-builder.test.js

```diff
@@ -3,65 +3,75 @@
+const DataSource = jdb.DataSource;
+const Memory = require('../lib/connectors/memory');
 
-var ModelDefinition = require('../lib/model-definition');
+const ModelDefinition = require('../lib/model-definition');
 
 describe('ModelBuilder class', function() {
-  var memory;
+  let memory;
   beforeEach(function() {
     memory = new DataSource({connector: Memory});
   });
 
   describe('Model inheritance', function() {
-    it('should inherit prototype using option.base', function() {
-      var modelBuilder = memory.modelBuilder;
-      var parent = memory.createModel('parent', {}, {
+    it('inherits prototype using option.base', function() {
+      const modelBuilder = memory.modelBuilder;
+      const parent = memory.createModel('parent', {}, {
         relations: {
           children: {
             type: 'hasMany',
             model: 'anotherChild',
           },
         },
       });
-      var baseChild = modelBuilder.define('baseChild');
+      const baseChild = modelBuilder.define('baseChild');
       baseChild.attachTo(memory);
       // the name of this must begin with a letter < b
       // for this test to fail
-      var anotherChild = baseChild.extend('anotherChild');
+      const anotherChild = baseChild.extend('anotherChild');
 
       assert(anotherChild.prototype instanceof baseChild);
     });
 
-    it('should ignore inherited options.base', function() {
-      var modelBuilder = memory.modelBuilder;
-      var base = modelBuilder.define('base');
-      var child = base.extend('child', {}, {base: 'base'});
-      var grandChild = child.extend('grand-child');
+    it('ignores inherited options.base', function() {
+      const modelBuilder = memory.modelBuilder;
+      const base = modelBuilder.define('base');
+      const child = base.extend('child', {}, {base: 'base'});
+      const grandChild = child.extend('grand-child');
       assert.equal('child', grandChild.base.modelName);
       assert(grandChild.prototype instanceof child);
     });
 
-    it('should ignore inherited options.super', function() {
-      var modelBuilder = memory.modelBuilder;
-      var base = modelBuilder.define('base');
-      var child = base.extend('child', {}, {super: 'base'});
-      var grandChild = child.extend('grand-child');
+    it('ignores inherited options.super', function() {
+      const modelBuilder = memory.modelBuilder;
+      const base = modelBuilder.define('base');
+      const child = base.extend('child', {}, {super: 'base'});
+      const grandChild = child.extend('grand-child');
       assert.equal('child', grandChild.base.modelName);
       assert(grandChild.prototype instanceof child);
     });
 
-    describe('merge policy with flag `configurableModelMerge`', function() {
+    describe('merge policy WITH flag `configureModelMerge: true`', function() {
       describe('ModelBaseClass.getMergePolicy()', function() {
-        const defaultMergePolicy = {
+        const legacyMergePolicy = {
```

Comment:
I will actually move the `ModelBaseClass.getMergePolicy()` block of test upper in the tests nesting levels as it is no more specific to a flag, but i kept this as is to ease your diff reading

I had to adapt a bit the tests when removing the feature flag